hibernate-core/src/test/java/org/hibernate/orm/test/entitygraph/EntityGraphAndJoinTest.java

```diff
@@ -112,7 +112,7 @@ private void executeQuery(SessionFactoryScope scope, boolean criteria, boolean l
 			final EntityGraph<?> entityGraph = session.getEntityGraph( "test-graph" );
 			final List<Person> resultList = query.setHint( HINT_SPEC_FETCH_GRAPH, entityGraph ).getResultList();
 			assertThat( resultList ).hasSize( 2 );
-			assertThat( resultList.stream().map( p -> p.getAddress().getId() ) ).containsExactly( 1L, 2L );
+			assertThat( resultList.stream().map( p -> p.getAddress().getId() ) ).contains( 1L, 2L );
```

Comment:
This changes test expectations -- and not just for GaussDB. Why?

Reply:
No order by, there is no guarantee of the data order. Table size, algorithms, indexes may lead to different data order.